independent-publisher-connectors/NFTMania/NFTMania.md

```diff
@@ -0,0 +1,38 @@
+**Proposal for NFTMania Connector**
```

Comment:
@shreyan1999 please remove this file. Thank you

Reply:
Okay. I will remove this file.